packages/rollup-plugin-import-meta-assets/src/rollup-plugin-import-meta-assets.js

```diff
@@ -24,25 +29,32 @@ function getRelativeAssetPath(node) {
  * Checks if a AST node represents this kind of expression: `new URL('./path/to/asset.ext', import.meta.url)`.
  *
  * @param {import('estree').Node} node - The AST node
- * @returns {boolean}
+ * @returns {false | 'static' | 'dynamic'}
```

Comment:
`false` doesn't play nicely with the new method name and purpose, instead use `undefined` when type can't be determined

Reply:
Will do

Reply:
thanks!